### PR TITLE
feat(TDP-3336): live datasets with TIC

### PR DESCRIPTION
**Pull request by skermabon**
**Link to the JIRA issue**
https://jira.talendforge.org/browse/TDP-3336

**Please check if the PR fulfills these requirements**
- [x] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [x] The code coverage on new code is > 75 % for backend and > 95% for frontend
- [x] The new code does not introduce new technical issues (sonar / eslint)
- [x] Functional tests have been performed
- [ ] Docker configuration files for config-std or config-cloud profiles are impacted

**Please check the browsers you've tested on**
- [x] Chrome, Firefox, Safari, Edge, IE11
- [ ] No, that's bad, this PR should not be merged !
- [ ] No, and no need to (backend changes only)

### CODE REVIEW

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/494/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/494/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/495/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/495/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/509/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/509/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/510/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/510/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/511/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/511/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/521/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/521/)

File: dataprep-backend-service/src/main/java/org/talend/dataprep/command/GenericCommand.java [hunk 1]
```diff
@@ -448,4 +448,13 @@ public String buildRequestReport(HttpRequestBase req, HttpResponse res) {
             return builder.toString();
         }
     }
+
+    public void setClient(HttpClient client) {
```
Comment (fhuaulme) [inline dataprep-backend-service/src/main/java/org/talend/dataprep/command/GenericCommand.java]: Why do you need a setter for this field?

Comment (skermabon) [inline dataprep-backend-service/src/main/java/org/talend/dataprep/command/GenericCommand.java]: Just for test. I will remove this method and use ReflectionTestUtils in tests

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/530/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/530/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/534/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/534/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/589/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/589/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/622/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/622/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/623/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/623/)

**Pull request merged**